testsuite/features/build_validation/migration/sle15sp3_ssh_minion_migration.feature

```diff
@@ -7,6 +7,21 @@ Feature: Migrate a SLES 15 SP3 Salt SSH minion to 15 SP4
   Scenario: Log in as admin user
     Given I am authorized for the "Admin" section
 
+  # pristine images for SSH minions use an old version of OS Salt
+  # Product Migration will be possible only if we update to latest
```

Comment:
Why we don't need the same scenario in a regular Salt Minion?

Reply:
TBH, I am puzzled by that as well, but it seems regular Salt minions are all using the latest venv-salt.
I don't know if that is a just a coincidence or we are actively doing something that keeps it updated in Sumaform (or the testsuite).

Reply:
Latest venv-salt-minion it's expected, we are installing it through cloud-init phase.
But I'm talking about classic salt (aka OS salt)

Reply:
Having OS Salt up to date is not needed on the regular minions then, as the Product Migration correctly detects that they are using venv-salt and it is the latest version available.

SSH Salt minions have only an old version of OS Salt installed, that's the issue with them.

Reply:
Oh right, sorry, I was thinking on Salt migration test case, not product migration.
All good, approving :)